aptos-move/framework/aptos-framework/sources/account.move

```diff
@@ -308,12 +308,17 @@ module aptos_framework::account {
     /// Basic account creation methods.
     ///////////////////////////////////////////////////////////////////////////
 
-    /// A resource account is used to manage resources independent of an account managed by a user.
-    public fun create_resource_account(source: &signer, seed: vector<u8>): (signer, SignerCapability) {
-        let bytes = bcs::to_bytes(&signer::address_of(source));
+    /// This is a helper function to compute resource addresses. Computation of the address
+    /// involves the use of a cryptographic and should be use thoughtfully.
+    public fun create_resource_address(source: &address, seed: vector<u8>): address {
```

Comment:
Why is source &address and not just address?

Reply:
we need to eventually pass in the reference to the bcs, we could theoretically make this a copy but that's just ... make a copy -> ref, when we could just push the ref requirement up. :shrug: